guides/xspec-setup.md

```diff
@@ -117,6 +120,48 @@ docker run \
     dustpancake/heasoft:6.30.1 tcsh
 ```
 
+## Calibration and CALDB
+
+In order to run some of the mission-specific HEASoft tasks you will require calibration files for those missions (e.g., for *NuSTAR* or *NICER*) as well as some mission non-specific general calibration information.
+
+Instructions and more information on the [The HEASARC Calibration Database](https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/caldb_intro.html) web page.
+
+For specific missions see the [Latest CALDB Files for Supported Missions](https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/caldb_supported_missions.html) web page.
+
+### General calibration
+
+Follow the [How to Install a Calibration Database](https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/install.html) instructions.
+
+Update `/Users/your_path/...` to the appropriate path. The following was up-to-date as of 2023-01-20.
+
+```
+export CALDB=/Users/your_path/caldb
+cd $CALDB
+wget https://heasarc.gsfc.nasa.gov/FTP/caldb/software/tools/caldb_setup_files.tar.Z
+tar -zxvf caldb_setup_files.tar.Z
+wget https://heasarc.gsfc.nasa.gov/FTP/caldb/data/gen/goodfiles_gen_20200625.tar.Z
+tar -zxvf goodfiles_gen_20200625.tar.Z
+```
+
+Edit the path in ``caldbinit.sh`` then to set up CALDB use the following
+
+```
+export CALDB=/Users/your_path/caldb
+source $CALDB/software/tools/caldbinit.sh
+```
+
+### NuSTAR calibration
+
+Up-to-date as of 2023-01-20. See [Latest CALDB Files for Supported Missions](https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/caldb_supported_missions.html). Note that at the moment a patch file is required for NuSTAR.
+
+```
+cd $CALDB
+wget https://heasarc.gsfc.nasa.gov/FTP/caldb/data/nustar/fpm/goodfiles_nustar_fpm_20221229.tar.gz
+tar -zxvf goodfiles_nustar_fpm_20221229.tar.gz
+wget https://heasarc.gsfc.nasa.gov/FTP/caldb/data/nustar/fpm/goodfiles_nustar_fpm_clockcor_20221229.tar.gz
+tar -zxvf goodfiles_nustar_fpm_clockcor_20221229.tar.gz
+```
+
```

Comment:
Also added notes about installing the calibration databases and files for specific missions.